packaging/rpm/postinst

```
@@ -1,19 +1,19 @@
 #!/bin/sh
-set -e
+# set -e
```

nit: Can you add a comment as to why we are ignoring errors and not failing?

---

Done